formats/polus-ome-zarr-converter-plugin/run-plugin.sh

```diff
@@ -6,13 +6,18 @@ datapath=$(readlink --canonicalize ../../data)
 # Inputs
 inpDir=/data/images/MaricRatBrain2019/standard/intensity
 filePattern="S1_R1_C1-C11_A1_c0{c}0.ome.tif"
-
+#OR
+filePattern="S1_R1_C1-C11_A1_c0{c}0.ome.zarr"
+fileExtension = ".ome.tif" 
+#OR
+fileExtension = ".ome.zarr" 
```

Comment:
I think this should be set up as a corresponding set of conditions, and comment out the sections that you aren't using.

For example:
```shell
# convert from ome.tif to ome.zarr
filePattern="S1_R1_C1-C11_A1_c0{c}0.ome.tif"
fileExtension=".ome.zarr"

# convert from ome.zarr to ome.tif
# filePattern="S1_R1_C1-C11_A1_c0{c}0.ome.zarr"
# fileExtension=".ome.tif" 
```

Reply:
I have addressed this in a new PR